projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -430,6 +431,7 @@ public static function load_etk_features() {
 			require_once __DIR__ . '/features/wpcom-documentation-links/wpcom-documentation-links.php';
 			require_once __DIR__ . '/features/wpcom-global-styles/index.php';
 			require_once __DIR__ . '/features/wpcom-legacy-fse/wpcom-legacy-fse.php';
+			require_once __DIR__ . '/features/wpcom-whats-new/wpcom-whats-new.php';
```

Comment:
We reverted too much! This should be removed since the files are no longer available.